### PR TITLE
Pyflakes cleanup

### PR DESCRIPTION
**Pull request by phlax**
ref #4239

### CODE REVIEW

File: pootle/apps/pootle_app/views/admin/__init__.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
...
 
-from .languages import *
-from .projects import *
-from .users import *
+from .languages import LanguageAdminView, LanguageAPIView
+from .projects import ProjectAdminView, ProjectAPIView
+from .users import UserAdminView, UserAPIView
+
+(LanguageAdminView, LanguageAPIView, ProjectAdminView, ProjectAPIView,
+ UserAdminView, UserAPIView)
```
Comment (unho) [inline pootle/apps/pootle_app/views/admin/__init__.py]: Not sure I like this kind of change.

Comment (phlax) [inline pootle/apps/pootle_app/views/admin/__init__.py]: import * is not explicit and is regarded as bad

https://www.python.org/dev/peps/pep-0008/#id17

Comment (unho) [inline pootle/apps/pootle_app/views/admin/__init__.py]: I am ok with explicit import, but is it necessary to have this tuple?

Comment (phlax) [inline pootle/apps/pootle_app/views/admin/__init__.py]: without it triggers a pyflakes unused local var error

Comment (dwaynebailey) [inline pootle/apps/pootle_app/views/admin/__init__.py]: Does feel like a cludge to get tests to pass.  I'm OK with that but can we at least be consistent in placement/use of tuple?

Comment (unho) [inline pootle/apps/pootle_app/views/admin/__init__.py]: At least put two blank lines after the imports.

File: pootle/apps/pootle_store/models.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 from django.contrib.auth import get_user_model
 from django.core.exceptions import ObjectDoesNotExist
 from django.core.urlresolvers import reverse
-from django.db import models, transaction
+from django.db import models
```
Comment (unho) [inline pootle/apps/pootle_store/models.py]: Can you please directly land the adding missing imports and remove unused imports changes?

File: pootle/apps/pootle_app/views/admin/__init__.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_app/views/admin/__init__.py]: > Does feel like a cludge to get tests to pass. I'm OK with that but can we at least be consistent in placement/use of tuple?

Yep it is a kludge but its kinda *the* kludge. We could use the __all__ var ie:

```
from foo import ClassForReexport, OtherClassForReexport


__all__ = [ClassForReexport, OtherClassForReexport]
```
but its really only a cosmetic difference

Comment (phlax) [inline pootle/apps/pootle_app/views/admin/__init__.py]: i think using `__all__` would serve the purpose of shutting codacy up at least 8/

Comment (unho) [inline pootle/apps/pootle_app/views/admin/__init__.py]: +1 on using `__all__` as long it doesn't introduce any other issue.

**Comment by dwaynebailey** [comment]
@phlax seriously I'd just have landed the missing and uneeded imports, they really don't need a PR.  Can we please have proper commit messages.  This introduced ``__all__ `` but does nothing to explain why.  Leaving it to some sucker at some unknown future date left wondering why.

The ``__all__`` still feels like a real kludge but I'm OK with it landing.

GTM with an actual human facing commit message.

**Pull request merged**